### PR TITLE
`va(` searches forward

### LINKED ISSUE #8024: `va(` not working
First of all, thanks for the awesome plugin 👍🏻

The issue is that in my case `va(` not working like in vim or nvim. For example: `myFunctionName(arg1,arg2)` and if positioned at `F` in normal mode, by pressing `va(` one would expect to be selected: `(arg1,arg2)`, but nothing happens. How can I fix this? Thank you very much 🙏🏽

### PR DESCRIPTION
**Pull request by smallkirby**
**What this PR does / why we need it**:

In current impl, `va(` searches only backward for an opening character.

```txt
// If positioned as `te|st`, `vi(d` would delete `test`.
// But if positioned as `|function`, `vi(d` would delete only `f`
function (test) {
  hoge
}
```

As far as I checked, Vim 8.2 searches forward for an opening char if it can't find it backward. So I fixed to behave the same.

After this PR, `va(`-family would work as below:

```txt
// If positioned as `te|st`, `vi(d` would delete `test`.
// And if positioned as `|function`, `vi(d` would also delete `test`
function (test) {
  hoge
}
```

**Which issue(s) this PR fixes**

- Fixes #8024 

**Special notes for your reviewer**:

- I didn't touch behavior of indent block and tag block, because vanilla Vim behaves as same already.

### CODE REVIEW

File: src/actions/motion.ts [hunk 1]
```diff
@@ -1860,7 +1860,20 @@ export abstract class MoveInsideCharacter extends ExpandingSelection {
...
     let openPos = PairMatcher.nextPairedChar(selStart, closingChar, vimState, true);
     if (openPos === undefined) {
-      return failedMovement(vimState);
+      // If opening character not found, search forwards
+      let lineNum = selStart.line;
+      while (true) {
+        if (lineNum >= vimState.document.lineCount) {
+          break;
+        }
+        const lineText = vimState.document.lineAt(lineNum).text.split('');
```
Comment (J-Fields) [inline src/actions/motion.ts:1869]: Isn't the `split()` unnecessary?

Comment (smallkirby) [reply src/actions/motion.ts]: You're right, fixed.

**Comment by smallkirby** [comment]
Thanks for your reveiw.

- I checked the comment and fixed.
- I fixed a bug in my commit.

**Comment by J-Fields** [comment]
Thanks!

**Pull request merged**